joomla/joomla-cms
# [5.3] Remove the not needed getPathway in the SiteApplication class

Pull Request for Issue #41653 .

### Summary of Changes

Removed the function, the parent class is doing the same and the params are not used.

### Testing Instructions

Check a site with a pathway/breadcrumbs applying the patch should change nothing 

### Link to documentations
Please select:
